packages/strapi-design-system/src/DateTimePicker/DateTimePicker.js

```diff
@@ -118,6 +118,8 @@ export const DateTimePicker = ({
         <Stack horizontal spacing={2}>
           <DatePicker
             data-testid="datetimepicker-date"
+            // DateTimePicker sharing its Field id to DatePicker component
+            parentFieldId={generatedId}
```

Comment:
What do you think about just passing `id` instead? This way we could avoid adding a new prop entirely ... 🤔 

```
<Datepicker id={generatedId} />
```

Reply:
mmh when `DatePicker` is used by itself, it needs to generate it's own `id` to pass to `TextInput`

if I pass the `DateTimePicker` `generatedId` to `DatePicker` through `id`, and then do:
```js
const generatedId = useId('datepicker', id);
```

`DateTimePicker` id and `DatePicker` would still be different 
- `DateTimePicker` id would be `datetime-label-1`
- `DatePicker` id would be `datepicker-datetime-label-1`

or I would need to remove the prefix `datepicker` when generating the id in `DatePicker` component - but would it mean risking id duplicates? 👀

Reply:
Apologies. I didn't see this was waiting for me. I think it would be ok to remove the prefix from `DatePicker`, because `useId` uses a singleton pattern and therefore generates a new id for every call. Also the react@18 hook doesn't support / use prefixes so we should start to fade them out I think ...